syntax = "proto3";

package prescan.common;


message Header {
  // Message publishing time in seconds.
  double timestamp_sec = 1;

  // Module name.
  string module_name = 2;

  // Sequence number for each message. Each module maintains its own counter for
  // sequence_num, always starting from 1 on boot.
  uint32 sequence_num = 3;

  // data version
  uint32 version = 4;

  string frame_id = 5;
}
